responsibleai/responsibleai/_managers/error_analysis_manager.py

```diff
@@ -273,6 +281,15 @@ def get(self):
         """
         return self._ea_report_list
 
+    def _get_error_analysis_schema(self):
+        """Get the schema for validating the error analysis output."""
+        schema_directory = (Path(__file__).parent.parent / '_tools' /
```

Comment:
should these strings be moved to some constants file?

Reply:
I see the same pattern in other files as well. https://github.com/microsoft/responsible-ai-widgets/blob/212dc5a168a196ab71ea7a29b60b0751ca301509/responsibleai/responsibleai/_managers/counterfactual_manager.py#L378. Will probably keep like others for now.